react-client/src/components/ClickButtons.jsx

```diff
@@ -13,13 +15,14 @@ class ClickButtons extends React.Component {
             liked: false,

```

Comment:
Are you trying to make these into a state?
I believe you need  a super as well?